pkgs/tools/inputmethods/kime/default.nix

```diff
@@ -0,0 +1,120 @@
+{ lib, stdenv, rustPlatform, rustc, cargo, fetchFromGitHub, pkg-config, cmake, extra-cmake-modules, llvmPackages, bash
+, withWayland ? true
+, withIndicator ? true, dbus ? null, dbus_libs ? null, libdbusmenu ? null
+, withXim ? true, xorg ? null, xlibs ? null, cairo ? null
+, withGtk2 ? true, gtk2 ? null
+, withGtk3 ? true, gtk3 ? null
+, withQt5 ? true, qt5 ? null
```

Comment:
```suggestion
, withIndicator ? true, dbus, dbus_libs, libdbusmenu
, withXim ? true, xorg, xlibs, cairo
, withGtk2 ? true, gtk2
, withGtk3 ? true, gtk3
, withQt5 ? true, qt5
```

Reply:
Build failed after remove `? null` :confused:

Reply:
Then there is something wrong with the expression which was hidden by ? null.

Reply:
How can I test build correctly? I tried with `nix-build -A kime` and it succeed but CI was failed

Reply:
xlib is an alias which is not allowed to be used in nixpkgs.